person.rb

```diff
@@ -6,6 +6,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- [ ] You should have `A method for adding rentals `, and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association Please add it, it may look like this 👇 

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
```